source/spp/mhq/spp_001_nl_mhq_heide/05_kwaliteitszorg.Rmd

```diff
@@ -0,0 +1,19 @@
+# Extra aspecten van kwaliteitszorg t.o.v. de subprotocols
+
+```{=html}
+<!--
+Geen herhaling van wat al in subprotocols staat
+-->
+```
+Enkele tips bij het maken van een vegetatie-opname:
+
+-   vertrappel de vegetatie in de plot zo min mogelijk vooraleer je een vrij volledige soortenlijst hebt en de bedekkingen hebt ingeschat. Doe hierna nog een check op soorten die je eventueel miste meer centraal in de plot of vanop de knieën
+-   als je na 5 minuten geen nieuwe soort meer vindt, dan wordt de opname stopgezet.
+-   maak eerst een lijst van de aanwezige soorten, schat pas bedekkingen als je lijst min of meer volledig is
+-   tracht de meest bedekkende soorten bovenaan te groeperen in je lijst
+-   in een 3mx3m plot is een 30cmx30cm vlakje daarbinnen gelijk aan 1%. Een band van 30 cm op een 3m is dan gelijk aan 10%
+-   de totale som van alle bedekkingen in éénzelfde vegetatielaag kan meer dan 100% zijn door overlap, maar kan ook erg laag zijn bijvoorbeeld 10% in buntgrasvegetatie op stuifduin.
+-   de totale som van kruidlaag, open bodem, strooisellaag, (korst)moslaag is minimaal 100%
+-   kiemplanten van bomen en struiken : naam laten volgend door "(k)"
+-   soorten net buiten opnamevlak: kunnen vermeld worden maar duidelijk weergeven door de naam te laten volgen door "(R)"
```

Comment:
```suggestion
-   Vertrappel de vegetatie in de plot zo min mogelijk vooraleer je een vrij volledige soortenlijst hebt en de bedekkingen hebt ingeschat. Doe hierna nog een check op soorten die je eventueel miste meer centraal in de plot of vanop de knieën.
-   Maak eerst een lijst van de aanwezige soorten, schat pas bedekkingen als je lijst min of meer volledig is.
-   Als je na 5 minuten geen nieuwe soort meer vindt, dan wordt de opname stopgezet.
-   Tracht de meest bedekkende soorten bovenaan te groeperen in je lijst.
-   In een 3 m x 3 m plot is een 30 cm x 30 cm vlakje daarbinnen gelijk aan 1%. Een band van 30 cm op een 3 m is dan gelijk aan 10%.
-   De totale som van alle bedekkingen in éénzelfde vegetatielaag kan meer dan 100% zijn door overlap, maar kan ook erg laag zijn, bijvoorbeeld 10% in buntgrasvegetatie op stuifduin.
-   De totale som van kruidlaag, open bodem, strooisellaag en (korst)moslaag is minimaal 100%.
-   Kiemplanten van bomen en struiken duid je aan met een "(k)" na de soortnaam.
-   Soorten net buiten opnamevlak kunnen vermeld worden, maar dit moet je duidelijk weergeven door de naam te laten volgen door "(R)".
```

Reply:
Oke Toon, bedankt voor de review, het lijkt mij goed om je inhoudelijke opmerkingen meteen ook te verwerken in deze versie, ik ga daar nu mee aan de slag ineens